.kuzzlerc

```diff
@@ -118,9 +118,11 @@
       "controllers": {
         "auth": {
           "actions": {
-            "login": true,
+            "logout": true,
```

Comment:
I think that allowing the `logout` action for `anonymous` users is meaningless, because this role is for unautenticated users...